rules/windows/network_connection/net_connection_win_malware_backconnect_ports.yml

```diff
@@ -13,7 +13,8 @@ tags:
 logsource:
     category: network_connection
     product: windows
-    definition: 'Use the following config to generate the necessary Event ID 10 Process Access events: <ProcessAccess onmatch="include"><CallTrace condition="contains">VBE7.DLL</CallTrace></ProcessAccess><ProcessAccess onmatch="exclude"><CallTrace condition="excludes">UNKNOWN</CallTrace></ProcessAccess>'
+   # definition: 'Use the following config to generate the necessary Event ID 10 Process Access events: <ProcessAccess onmatch="include"><CallTrace condition="contains">VBE7.DLL</CallTrace></ProcessAccess><ProcessAccess onmatch="exclude"><CallTrace condition="excludes">UNKNOWN</CallTrace></ProcessAccess>'
+    definition: 'Event ID 3 - Network Connection'
```

Comment:
No need for a definition for this kind of event. The previous definition was a typo.
```suggestion
```